carbon-design-system/carbon
# 15757 create a controlled accordion stories

Closes #15757 

Added controlled story in Accordion

**New**

- Added controlled story


#### Testing / Reviewing

1) Click on the Controlled story in Accordion.
2) Click on 'Expand All' button and it should open all Panels
3) Click on 'Collapse All' button and it should close all Panels
